src/internal/m365/collection/groups/export.go

```diff
@@ -198,3 +213,146 @@ func makeMinimumChannelMesasge(item models.ChatMessageable) minimumChannelMessag
 		Subject:              ptr.Val(item.GetSubject()),
 	}
 }
+
+//-------------------------------------------------------------
+// Conversation Posts
+//-------------------------------------------------------------
+
+// streamConversationPosts adds the post items into the export stream channel.
+func streamConversationPosts(
+	ctx context.Context,
+	drc []data.RestoreCollection,
+	backupVersion int,
+	cec control.ExportConfig,
+	ch chan<- export.Item,
+	stats *metrics.ExportStats,
+) {
+	defer close(ch)
+
+	errs := fault.New(false)
+
+	for _, rc := range drc {
+		ictx := clues.Add(ctx, "path_short_ref", rc.FullPath().ShortRef())
+
+		for item := range rc.Items(ctx, errs) {
+			itemCtx := clues.Add(ictx, "stream_item_id", item.ID())
```

Comment:
not much point in having both ctx vars, since only one is used.
```suggestion
		for item := range rc.Items(ctx, errs) {
			ictx := clues.Add(
				ctx,
				"path_short_ref", rc.FullPath().ShortRef(),
				"stream_item_id", item.ID())
```